lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -106,11 +106,10 @@
 def execute_show_command(command, module):
     transport = module.params['transport']
     if transport == 'cli':
```

Comment:
I would suggest removing the transport check (just so it is clear that the intention is to always ask for JSON), and, as we have this function gatekeeping all the `run_commands()` calls, only annotating the commands with `'output': 'text'` or `'output': 'json'` here. What is here works, but is inconsistent both in process (`'output': 'text'` vs `| json`) and location